### DISCUSSION TITLE
Can I define no go zones?

### DISCUSSION THREAD

**Discussion by schultza** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#249] [Q&A] [answered]
Can I add new no go zones via the card?

**Comment by PiotrMachowski** [comment, marked as answer]
No, it is not possible in current version

**Reply by schultza** [reply]
Thansk for your answer!

**Reply by PiotrMachowski** [reply]
Maybe someday ;)

**Reply by ScottG489** [reply]
Was there any progress on this?

Just curious because I know we can select zones to clean on the map so I would think we could get no-go zone coordinates the same way. I believe the underlying library supports adding no-go zones, correct?

**Reply by PiotrMachowski** [reply]
No, it's not a high priority improvement.

The problem with no-go zones is that they can also be rotated. I would also have to implement a dedicated flow and add it to every supported integration. Right now I focus on making card more easily configurable
